### DISCUSSION TITLE
oSC 24 is coming, let's meet and submit some talk!

### DISCUSSION THREAD

**Discussion by denisok** [openSUSE/openSUSEway#159] [General]
Lets discuss who comes to the conference, so we might have small meetup and who want to speak about openSUSEway on oSC24. So we can submit a talk[s].

https://events.opensuse.org/conferences/oSC24

**Comment by denisok** [comment]
I would like to submit about openSUSEway on aarch64, and it could be a part of other's presentation. 

We can do one big together with @FilippoBonazziSUSE who can speak about features, improvements and etc.

Lets discuss...

**Comment by FilippoBonazziSUSE** [comment]
I will be at the conference, and it would be cool to present new features and new tools. Talks like these are always cool to attend, so it should be a good talk to give

**Comment by denisok** [comment]
talk will be here: https://github.com/openSUSE/openSUSEway/tree/main/docs/conferences/oSC24
